META6.json

```diff
@@ -1,5 +1,5 @@
 {
-  "perl"        : "6.*",
+  "perl"        : "6.d",
   "name"        : "Method::Modifiers",
   "version"     : "*",
```

Comment:
@jjatria I think the error complains about the asterisk in "version" key?

Reply:
Ah, you're absolutely right! :sweat_smile: 
I'll pus an update for this as soon as I have a minute

Reply:
Fixed. I've set 0.1.0 as the version for now, but that can be modified.